test/fixtures.js

```diff
@@ -1,33 +1,66 @@
 const nock = require('nock');
+const url = require('url');
 
 /**
  * This is a fixture module for telescope tests which contains measures to
  * maintain a reproducible environment for system testing
  */
 
-const validBody =
+const getAtomUri = () => 'https://test321.blogspot.com/feeds/posts/default/-/open-source';
+const getRssUri = () => 'https://test321.blogspot.com/feeds/posts/default/-/open-source?alt=rss';
+const getHtmlUri = () => 'https://test321.blogspot.com/blog';
+// Remove leading protocol from a URI
+const stripProtocol = uri => uri.replace(/^https?:\/\//, '');
+
+const getFeedBody = () =>
   '<?xml version="1.0" encoding="UTF-8" ?><rss version="2.0"><channel><title>W3Schools Home Page</title><link>https://www.w3schools.com</link><description>Free web building tutorials</description><item><title>RSS Tutorial</title><link>https://www.w3schools.com/xml/xml_rss.asp</link><description>New RSS tutorial on W3Schools</description></item><item><title>XML Tutorial</title><link>https://www.w3schools.com/xml</link><description>New XML tutorial on W3Schools</description></item></channel></rss>';
```

Comment:
Is it possible to fold this into multiple lines with max line width=128 characters? It will greatly help with code readability.